tools/tsp-client/README.md

```diff
@@ -2,45 +2,65 @@
 
 A simple command line tool for generating TypeSpec clients.
 
-### Usage
+### Installation
 ```
-tsp-client [options] <outputDir>
+npm install @azure-tools/typespec-client-generator-cli
 ```
 
-## Options
-
-### <outputDir> (required)
-
-The only positional parameter. Specifies the directory to pass to the language emitter.
-
-### --emitter, -e (required)
-
-Specifies which language emitter to use. Current choices are "csharp", "java", "javascript", "python", "openapi".
+### Usage
+```
+tsp-client <command> [options]
```

Comment:
I would go with an even shorter name.  How about `tspc`?  The TypeSpec compiler itself is `tsp`, and we currently use `tsv` for "TypeSpec Verification" (though I might be willing to change to `tspv`, if we want a suite of tools named `tsp_`.

Reply:
If the package name is `@azure-tools/typespec-client-generator`, another option for the command name could be `tspcg`.

Reply:
`tspc` could be an option. Thoughts @lmazuel, @m-nash, @allenjzhang?

Reply:
Or even `tcg` for `typespec-client-generator`?  I like relaly short CLI names.

Reply:
So part of the original idea behind naming this `tsp-client` was to make it easier for when we ideally add it into the `tsp` cli, to make the transition easier so we go from `tsp-client` to `tsp client`. 

I think it is important to keep `tsp` as the beginning of the command. So in that sense, I'm ok with `tspc` to be the shorter option, keeping in mind that in the future we might have to replace other tools that use it with `tsp client`.

Reply:
If we think this is a temporary name until we use `tsp client`, then I agree `tsp-client` may be the best name.

Reply:
Ok let's keep it like this since we would like to move this under `tsp` eventually.